app/imports/ui/pages/TestStuff.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Container, Header, Button } from 'semantic-ui-react';
+import { postTest } from '../../api/solc/connect2Compiler';
+import { deployContract } from '../../api/ethers/ethersFunctions';
+
+let contracts = [];
+
+/** Test page to compile a smart contract with the python script and */
+class TestStuff extends React.Component {
```

Comment:
I think we should avoid usage of `Stuff` . Maybe `TestContract` would be better.